### PR TITLE
Fix ESLint errors in cvat-core/src/server-proxy.ts

### PR DESCRIPTION
**Pull request by SpecLad**
Specifically:

* `padded-blocks`
* `@typescript-eslint/no-var-requires`
* `@typescript-eslint/return-await`

### Motivation and context

As far as I know, "go to definition" in VS Code only works with ES6-style imports. Since I was examining code in this file, I updated the imports to make it easier. I fixed the other ESLint errors, because otherwise it wouldn't let me commit the code. 🙂 

### How has this been tested?

I'm relying on CI for this.

### Checklist

- [x] I submit my changes into the `develop` branch
- ~~[ ] I have added a description of my changes into [CHANGELOG](https://github.com/cvat-ai/cvat/blob/develop/CHANGELOG.md) file~~
- ~~[ ] I have updated the [documentation](
  https://github.com/cvat-ai/cvat/blob/develop/README.md#documentation) accordingly~~
- ~~[ ] I have added tests to cover my changes~~
- ~~[ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))~~
- [x] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/cvat-ai/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/cvat-ai/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/cvat-ai/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/cvat-ai/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/cvat-ai/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

File: cvat-core/src/server-proxy.ts [hunk 1]
```diff
@@ -742,7 +755,7 @@ class ServerProxy {
                 }
             }
             try {
-                return wait();
+                return await wait();
```
Comment (bsekachev) [inline cvat-core/src/server-proxy.ts:758]: We do not need ``await`` when return result of an async function from an async function..

File: cvat-core/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cvat-core",
-  "version": "7.0.1",
+  "version": "7.0.2",
```
Comment (bsekachev) [inline cvat-core/package.json:3]: If it is only refactoring, we do not need to update version number

File: cvat-core/src/server-proxy.ts (hunk 1 above)
Comment (SpecLad) [reply cvat-core/src/server-proxy.ts:758]: In this case we do need `await`. Without it, the function will immediately return the promise it gets from `wait`, and the `catch` block will never be entered.

File: cvat-core/package.json (hunk 2 above)
Comment (SpecLad) [reply cvat-core/package.json:3]: The `return await` change is technically a behavior change.

**Comment by SpecLad** [comment]
I had to fix a bug in two other files for the tests to pass, and _those_ files fail ESLint. I'm not going to fix them (at least for now), so the ESLint build job will be red.

**Pull request merged**